MailCore/Utils/MessageBodyUtils.swift

```diff
@@ -118,6 +145,85 @@ public enum MessageBodyUtils {
         }
     }
 
+    private static func createPrintHeader(message: Message) throws -> Element {
+        let rootHeaderDiv = try Element(Tag("div"), "").attr("id", "printHeader")
+        let firstSeparator = try Element(Tag("hr"), "")
+            .attr("size", "1")
+            .attr("color", "black")
+            .attr("width", "100%")
+            .attr("style", "margin-bottom: 10px")
+        let secondSeparator = try Element(Tag("hr"), "")
+            .attr("size", "1")
+            .attr("color", "LightGray")
+            .attr("width", "100%")
+            .attr("style", "margin-top: 10px")
```

Comment:
Is it possible to use only one style attribute ? Something like `.attr("style", "margin-top: 10px; margin-bottom: 10px")`